src/test/debuginfo/vec-slices.rs

```diff
@@ -53,6 +53,11 @@
 // gdb-command:print *((int64_t[2]*)('vec_slices::MUT_VECT_SLICE'.data_ptr))
 // gdb-check:$15 = {64, 65}
 
+//gdb-command:print mut_slice.length
```

Comment:
But that is done in the second test, right? I looked at the buildbot logs and noticed that's exactly where the build was failing:
```

---- [debuginfo-gdb] debuginfo-gdb/vec-slices.rs stdout ----
	
error: line not found in debugger output: $17 = {1, 2, 3, 4, 5}
status: exit code: 0
command: arm-linux-androideabi-gdb -quiet -batch -nx -command=x86_64-unknown-linux-gnu/test/debuginfo-gdb/vec-slices.debugger.script
stdout:
------------------------------------------
```